docs/source/conf.py

```diff
@@ -193,7 +193,7 @@
 intersphinx_mapping = {'python': ('http://docs.python.org/2/', None),
                        'rpy2': ('http://rpy.sourceforge.net/rpy2/doc-2.4/html/', None),
                        'traitlets': ('http://traitlets.readthedocs.org/en/latest/', None),
-                       'jupyterclient': ('http://jupyter-client.readthedocs.org/en/latest/', None),
+                       'jupyter-client': ('http://jupyter-client.readthedocs.org/en/latest/', None),
```

Comment:
Please don't change this, there's already one reference using it as `jupyterclient`. Also, Sphinx gives a warning if it contains a dash, though it doesn't seem to break anything.